doc/modules/ROOT/pages/projects.adoc

```diff
@@ -163,7 +163,7 @@ What this does is:
 
 . add your own type of project, in this case `npm` package.
 . add a list of files and/or folders in a root of the project that helps to identify the type, in this case it is only `package.json`.
-. add _project-file_, which is typically the primary project configuration file. In this case that's `package.json`.
+. add _project-file_, which is typically the primary project configuration file. In this case that's `package.json`. The value can contain wildcards and/or be a list containing multiple project files to look for.
```

Comment:
I'd also add the C# example in the same project definitions (there are some about Ruby projects if I recall correctly), so that's even clearer.

Reply:
I've added the C#/F# registration to the docs in that section and explained its structure as an example. It does use the function form for `MARKER-FILES`, but I just left that opaque.

I also added to the docs that using a function for markers is actually supported, which was implemented a long time ago but didn't seem to be in the docs.